src/overlays/actors/ovl_En_Time_Tag/z_en_time_tag.h

```diff
@@ -7,10 +7,57 @@ struct EnTimeTag;
 
 typedef void (*EnTimeTagActionFunc)(struct EnTimeTag*, PlayState*);
 
-#define ENTIMETAG_GET_1F(thisx) ((thisx)->params & 0x1F)
-#define ENTIMETAG_GET_SWITCHFLAG(thisx) ((thisx)->params & 0x7F)
-#define ENTIMETAG_GET_1FE0(thisx) (((thisx)->params & 0x1FE0) >> 0x5)
-#define ENTIMETAG_GET_E000(thisx) (((thisx)->params & 0xE000) >> 0xD)
+#define TIMETAG_GET_TYPE(thisx) (((thisx)->params & 0xE000) >> 0xD)
+
+#define TIMETAG_GET_KICKOUT_TEXT_TYPE(thisx) (((thisx)->params & 0x1FE0) >> 0x5)
+#define TIMETAG_GET_KICKOUT_EXIT_INDEX(thisx) ((thisx)->params & 0x1F)
```

Comment:
```suggestion
#define TIMETAG_KICKOUT_GET_TEXT_TYPE(thisx) (((thisx)->params & 0x1FE0) >> 0x5)
#define TIMETAG_KICKOUT_GET_EXIT_INDEX(thisx) ((thisx)->params & 0x1F)
```
For these subtypes, I think it would be better to have the subsystem as the second word. Same for the GET_SOARING and GET_DIARY types